docs: initial content for Debugging

Ombuweb opened the pull request.

ErjanGavalji commented on location.md:
```diff
@@ -0,0 +1,12 @@
+
+Location module is designed to be a wrapper of all available platforms APIs (related to location), which simplifies the process of creating a location powered applications with NativeScript framework. NativeScript location module uses an accuracy criteria approach to deliver locations. Which means that getting a location (or monitoring locations) is powered by the best appropriate location provid ...
+
+More information about location module can be found in [API-Ref](../Content/ApiReference/location/location.md) and [How-to](../Content/ApiReference/location/HOW-TO.md) pages.
```
We should not have the `Content` folder.
Please, make the links like

 [API-Ref](./ApiReference/location/location.md)

hshristov commented on location.md:
```diff
@@ -0,0 +1,12 @@
+
+# Location
+
+Location module is designed to be a wrapper of all available platforms APIs (related to location), which simplifies the process of creating a location powered applications with NativeScript framework. NativeScript location module uses an accuracy criteria approach to deliver locations. Which means that getting a location (or monitoring locations) is powered by the best appropriate location provid ...
```
I think this applies to all modules so we could skip/remove it :)

(for example if you need a high accuracy GPS provider will be used) - not clear.
WI-FI - should be Wi-Fi (or WiFi)

Ombuweb committed:
docs: initial content for Debugging

rigor789 committed:
Merge branch 'main' into docs/debugging

rigor789 committed:
Merge branch 'main' into docs/debugging

rigor789 committed:
chore: move to /guide/debugging

Shorter URL feels cleaner, and we don't have to stick to the same folder structure as the sidebar entries.

rigor789 committed:
wip: restructuring and rewording

rigor789 commented:
I'm working on restructuring and rewording the debugging docs a little - moved this back to `in progress`, I'll be pushing additional commits as I go.

rigor789 committed:
Update debugging.md

rigor789 committed:
Update debugging.md

rigor789 committed:
simplify console section

rigor789 committed:
Update debugging.md

rigor789 committed:
cleanup

rigor789 committed:
Update debugging.md
